proto/rove.proto

```diff
@@ -62,18 +60,26 @@ message ValidateRequest {
   optional string extra_spec = 10;
 }
 
-message TestResult {
-  google.protobuf.Timestamp time = 1;
+message FlagSeries {
   // data source defined identifier, it's recommended to use this to identify
   // a timeseries/station/location as appropriate
-  string identifier = 2;
-  Flag flag = 3;
+  string id = 1;
+  // flags indicating the results of running the check on each element of the
+  // given timeseries. The time in the timeseries each flag applies to is 
+  // `start_time` + (`time_resolution` * index), where `start_time` and
+  // `time_resolution` are arguments given in the `ValidateRequest`
+  repeated Flag flags = 2;
```

Comment:
Should we also return `start_time` and `time_resolution`?

Reply:
I'm open to changing this, but I figured there was no point adding them since the caller already has them? It's just us cloning, serialising, and transferring extra data when the caller can just reference it locally? Granted it's not a lot of data though

Reply:
Yeah, probably not really necessary

Reply:
I guess it's also worth clarifying: are you suggesting returning them from validate_direct as well, or just the gRPC API?